thredds/casestudies/catalog.xml

```diff
@@ -344,5 +345,22 @@
       </filter>
     </datasetScan>
 
+    <dataset name="Moisture Transport" ID="NamAggregate" urlPath="nam/Agg.nc">
```

Comment:
I'm not sure if we doing it consistently but maybe its nice to put "casestudies" in the ID and urlPath

Reply:
Done :)